src/lib.rs

```diff
@@ -45,3 +47,23 @@ mod log {
   pub(crate) use debug as trace;
   pub(crate) use debug as warn;
 }
+
+pub enum BuildId<'a> {
```

Comment:
Can we add some documentation for this type? A doc test may also be helpful to understand what is what.